File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Vision/Models/SlotType.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Vision.Models
{
    public class SlotType
    {
        [XmlAttribute(AttributeName = "id")]
        public string Id { get; set; }

        [XmlElement(ElementName = "description")]
        public string Description { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/vision/models/appointments/SlotType.kt
package mocking.vision.models.appointments

import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement

@Suppress("UnusedPrivateMember", "Required for xml serialisation")
data class SlotType(
        @XmlAttribute(name = "id") private var id: Int?,
        @XmlElement(namespace = "urn:vision")
        var description: String
)
